src/uptdd.f

```diff
@@ -3,16 +3,21 @@
 C>
 C> @author J. Woollen @date 1994-01-06
 
-C> This subroutine returns the bit-wise representation of the FXY value corresponding to, sequentially,
-C> a particular (IENT'th) "child" mnemonic of a Table D sequence ("parent") mnemonic.
+C> Returns the integer FXY value corresponding to a child
+C> mnemonic of a Table D sequence parent mnemonic.
```

Comment:
We've always taken care to explicitly call this the "bit-wise representation of the FXY value", rather than just an "integer FXY value". 

For example, if the character representation of the descriptor is "063022", then one could wrongly assume that the associated "integer FXY value" is just 63022, but instead what we're really talking about here is a value of 16150.  See the remarks in the docblock for ifxy() for a detailed example.

Bottom line, there's a reason we've always referred to the latter value as the "bit-wise representation of the FXY value" throughout the entire library, rather than just  the "integer FXY value" :-)